python/templates/Object.h.jinja2

```diff
@@ -43,6 +43,8 @@ public:
   using collection_type = {{ class.bare_type }}Collection;
 
 {{ macros.constructors_destructors(class.bare_type, Members) }}
+  /// converting constructor from mutable object
+  {{ class.bare_type }}(const Mutable{{ class.bare_type }}& other);
```

Comment:
If we mark this `explicit` things do not work any longer, right? Because then we lose the implicit conversion that we rely on in many places, I suppose.

Reply:
Exactly, plenty of 'no known conversion' errors